opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -914,9 +909,14 @@ namespace Opm {
                     std::string msg = "Iter";
 
                     std::vector< std::string > key( numComp );
-                    for (int phaseIdx = 0; phaseIdx < numPhases(); ++phaseIdx) {
-                        const std::string& phaseName = FluidSystem::phaseName(flowPhaseToEbosPhaseIdx(phaseIdx));
-                        key[ phaseIdx ] = std::toupper( phaseName.front() );
+                    for (unsigned phaseIdx = 0; phaseIdx < FluidSystem::numPhases; ++phaseIdx) {
+                        if (!FluidSystem::phaseIsActive(phaseIdx))
+                            continue;
+
+                        const unsigned canonicalCompIdx = FluidSystem::solventComponentIndex(phaseIdx);
```

Comment:
Again the solventComponentIndex?

The change from associating the convergence display with phases to associating it with components is good and the correct approach.

Reply:
Same answer as above.